File: src/python/tutorias/domain/asignatura.py
class Asignatura:
    def __init__(self, id_asignatura: int, nombre: str, descripcion: str):
        self._id = id_asignatura
        self._nombre = nombre
        self._descripcion = descripcion

    def consultar_informacion(self) -> str:
        return f"{self._nombre}: {self._descripcion}"

File: src/java/edu/uees/tutorias/domain/Asignatura.java
package edu.uees.tutorias.domain;

public class Asignatura {

    private int id;
    private String nombre;
    private String descripcion;

    public Asignatura(
            int id,
            String nombre,
            String descripcion) {

        this.id = id;
        this.nombre = nombre;
        this.descripcion = descripcion;
    }

    public String consultarInformacion() {
        return nombre + ": " + descripcion;
    }
}